core-bundle/composer.json

```diff
@@ -94,13 +95,13 @@
         "scheb/2fa-trusted-device": "^6.0",
         "scrivo/highlight.php": "^9.18",
         "scssphp/scssphp": "^1.5",
-        "simplepie/simplepie": "^1.3",
```

Comment:
If we remove this dependency, we have to remove the *contao/modules/ModuleRssReader.php* file too I think because it is still used there.
But for better BC we probably want to keep `ModuleRssReader` and therefore should keep `simplepie` too?
Alternatively we could add a class_exists check to `ModuleRssReader` and throw a meaningful exception if `simplepie` is missing.

Reply:
Yeah, we probably have to keep the module and its dependencies around for a while now. As much as I would like to remove both I'd rather leave it as is until 6.0.

Reply:
I gave this some more thought: I would have to reverse much more if we want to keep ModuleRssReader, esp. column names. Currently all uses of the legacy RSS reader will be migrated to the new FeedReader so there kinda is a smooth migration path available, IF you use the core legacy module and template 🤔

Reply:
Honestly, the `ModuleRssReader` remains there now only for BC. And I'd argue that the use cases to extend from that were rather little. So I don't think there will be many cases out there. Mostly, people will just use that module and not even notice that the implementation in the background has changed :)
So I'd prefer to get rid of this dependency and add a `class_exists()` test in the `ModuleRssReader`. 
Also `ModuleRssReader` should  have a `trigger_deprecation()` call.

Reply:
Templates are not covered by the BC promise. And for the BC of the column names in `ModuleRssReader`: Maybe we can map them?

Reply:
See 
8e60be8